SPECS/binutils/binutils.spec

```diff
@@ -130,7 +150,33 @@ sed -i 's/testsuite/ /g' gold/Makefile
 %{_libdir}/libopcodes.a
 %{_libdir}/libopcodes.so
 
+%if %{with gprofng}
+%files gprofng
+%{_bindir}/gp-*
+%{_bindir}/gprofng
+%{_mandir}/man1/gp-*
+%{_mandir}/man1/gprofng*
+%dir %{_libdir}/gprofng
+%{_includedir}/sframe.h
+%{_includedir}/sframe-api.h
+%{_includedir}/collectorAPI.h
+%{_includedir}/libcollector.h
+%{_includedir}/libfcollector.h
+%{_libdir}/gprofng/*
+%{_libdir}/libsframe.a
+%{_libdir}/libsframe.so
+%{_libdir}/libsframe.so.0
+%{_libdir}/libsframe.so.0.*
+
+%dir %{_exec_prefix}/lib/debug/%{_libdir}/gprofng
+%{_exec_prefix}/lib/debug/%{_libdir}/gprofng/libgp*
+%{_sysconfdir}/gprofng.rc
+%endif
+
 %changelog
+* Wed Aug 30 2023 Sindhu Karri <lakarri@microsoft.com> - 2.40-1
```

Comment:
Also, `binutils` affects the `toolchain` so please update the `manifest` files as well.

Reply:
Updated manifests